internals/cli/export_test.go

```diff
@@ -17,20 +17,40 @@ package cli
 import (
 	"fmt"
 
+	"github.com/canonical/go-flags"
+
 	"github.com/canonical/pebble/client"
 )
 
 func RunMain() error {
-	_, ClientConfig.Socket = getEnvPaths()
-	return Run(&RunOptions{
-		ClientConfig: ClientConfig,
-	})
+	return Run(RunOptionsForTest())
+}
+
+func RunOptionsForTest() *RunOptions {
```

Comment:
Would have liked to call this `RunOptions()` (just like `Client()`), but it's already the name of the type.